tests/10-rdfc-create.js

```diff
@@ -78,6 +96,13 @@ describe('eddsa-rdfc-2022 (create)', function() {
           ).should.equal(true, 'Expected at least one proof to have "type" ' +
             'property value "Multikey".');
         });
+        /**
+         * @see {@link https://www.w3.org/TR/vc-di-eddsa/#dataintegrityproof}
```

Comment:
does your version of chrome have the highlight links?

Reply:
It is highlighted in the IDE